code-challenges.test.js

```diff
@@ -10,50 +10,159 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
-// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
-
-// a) Create a test with expect statements for each of the variables provided.
-
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
-
-// b) Create the function that makes the test pass.
-
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
+const object1 = { number: 15 };
 // Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
+const object2 = { number: 0 };
 // Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
+const object3 = { number: -7 };
 // Expected output: "-7 is not divisible by three"
 
+describe("checkDiv3", () => {
+  it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not", () => {
```

Comment:
Excellent work on this problem!!  To help keep the variables localized to the scope they are being used, you can move the test variables into the test itself right before the expect method.